config/shared/activities.js

```diff
@@ -111,3 +108,9 @@ function getRepositoryRelationships(schemaId) {
   return flatMap(structure, it => it.relationships)
     .reduce((acc, { type }) => union(acc, [type]), []);
 }
+
+function parseType(type) {
+  const schemaId = type.includes('/') && first(type.split('/'));
+  const typeName = type.includes('/') && type.split('/')[1];
+  return { schemaId, typeName };
```

Comment:
Umm, if it does not include slash return an empty object; oherwise just split and destructure array items forming the result.

Reply:
Something like this:
```js
function parseType(type) {
  const [schemaId, typeName] = type.split('/');
  return typeName ? { schemaId, typeName } : {};
}
```

Reply:
Be aware that if typeName contains slash you'll get only first part of it.